docs/utilities/Results.rst

```diff
@@ -298,4 +300,37 @@ Example of how this method can be used::
                 "labels_size": [("Origin", "Destination", "Time", "Size")],
                 }
 
-  plot_scatter(input_data, args)
\ No newline at end of file
+  plot_scatter(input_data, args)
+
+
+.. _results_is_feasible:
+
+Is Feasible
+------------
+
+**Method Description**
+
+Verifies the solution contained in a Pyomo model object is feasible. This requires iterating through all variables and constraints and ensuring that the constraint and variable bounds are satisfied at the solution present in the model.
+
+**How to Use**
+
+The method requres that a Pyomo model be passed as the first argument. Two other keyword arguments are optional:
+
+1. *"bound_tol"* - Violation tolerance for bounds. Default value is 1e-3.
+
+2. *"cons_tol"* - Violation tolerance for constraints. Default value is 1e-3.
+
+
+Example of how this method can be used::
+
+  from pareto.utilities.results import is_feasible, nostdout
+
+  # Assume model is any Pyomo model that has already been created and solved
+
```

Comment:
The "nostdout()" is a function that is defined in our code. While maybe self-explanatory, I suggest adding a comment
```suggestion
# With statement suppresses messages from the method call. 
```

Reply:
Good suggestion - addressed!